frontend/src/types/ReviewTypes.ts

```diff
@@ -0,0 +1,81 @@
+export type AuthorRequest = {
```

Comment:
nit: add documentation for all these types

Reply:
I feel like that would be excessive documentation? The field names are self-explanatory